src/Avalonia.Controls/Primitives/AccessText.cs

```diff
@@ -68,11 +69,15 @@ private protected override void RenderCore(DrawingContext context)
             if (underscore != -1 && ShowAccessKey)
             {
                 var rect = TextLayout!.HitTestTextPosition(underscore);
-                var offset = new Vector(0, -1.5);
+
+                var x1 = Math.Round(rect.Left, MidpointRounding.AwayFromZero);
```

Comment:
I think you can use Math.Floor here

Reply:
I used `Round` so that it aligns with the nearest pixel edge.  Otherwise, something like `rect.Left == 0.99` (effectively `1`) would render all the way to `0`.  But we can tweak it if you prefer that.

Reply:
Okay fine for me